.snippets/code/builders/interoperability/xcm/send-execute-xcm/execute/execute-with-polkadot.js

```diff
@@ -30,7 +30,7 @@ const instr2 = {
   },
 };
 const message = { V4: [instr1, instr2] };
-const maxWeight = { refTime: 400000000n, proofSize: 14484n };
+const maxWeight = { refTime: 7750000000n, proofSize: 4960000n };
```

Comment:
This is not correct, you should use the XCM Runtime API to get the values here.

Also 

```
const instr2 = {
  DepositAsset: {
    assets: { Wild: 'All' },
    beneficiary: {
      parents: 0,
      interior: {
        X1: [
          {
            AccountKey20: {
              key: '0x3Cd0A705a2DC65e5b1E1205896BaA2be8A07c6e0',
            },
          },
        ],
      },
    },
  },
};
```


Should actually be:

```
const instr2 = {
  DepositAsset: {
    assets: {
      Wild: {
        AllCounted: 1,
      },
    },
    beneficiary: {
      parents: 0,
      interior: {
        X1: [
          {
            AccountKey20: {
              key: '0x3Cd0A705a2DC65e5b1E1205896BaA2be8A07c6e0',
            },
          },
        ],
      },
    },
  },
};

```

Reply:
![image](https://github.com/user-attachments/assets/7bcb8d0d-e0c8-4625-a331-5df80d4a3ef9)